pages/linux/asterisk.md

```diff
@@ -0,0 +1,29 @@
+# asterisk
+
+> Telephone and exchange (phone) server.
+> This executable is used for running the server itself, and managing an already running instance.
+> More information: <https://wiki.asterisk.org/wiki/display/AST/Home/>.
+
+- Connect to a running server, and turn on some logging:
+
+`asterisk -r -vvv`
+
+- Show chan_SIP clients (phones):
+
+`asterisk -r -x "sip show peers"`
```

Comment:
Are there any long forms of these arguments? We generally prefer long forms on  tldr pages to aid clarity.

Reply:
Unfortunately not. I can see some other tldr pages uses syntax such as `[r]econnect`, would this work for an extra example such as this?

```markdown
- [r]econnect to a running server and e[x]ecute a command:

`asterisk -r -x "{{command}"`
```

Reply:
No problem - it doesn't hurt to check. Yeah, that extra example would be great if possible!

Reply:
How's that looking now?